### DISCUSSION TITLE
No thumbnails on mobile

### DISCUSSION THREAD

**Discussion by beingabstrac** [muxinc/player.style#155] [Q&A]
Not able to see thumbnails on mobile, works well on web. I'm using Microvideo.

**Comment by luwes** [comment]
thanks for the feedback @beingabstrac.

do you mean the timeline thumbnail previews? 
I just tested and on my iPhone they do seem to work at https://player.style/themes/microvideo?asset=portrait
it does require dragging your finger along the timeline

**Comment by beingabstrac** [comment]
![Image](https://github.com/user-attachments/assets/131363c3-d3bb-4baf-8061-567ad9f33fda)
Like it's just black, the preview/thumbnail shows only when I click play.

**Comment by beingabstrac** [comment]
Also is there a way to hide/remove the seekbar at the bottom in Microvideo?

**Comment by luwes** [comment]
it's possible on mobile the video will not preload so it won't show any content yet until the video is played. 
you'd need to set a poster image to get around that because the mobile device controls that behavior.

did you set a `poster` attribute on the video element?

or there is also the possibility to set a slotted poster element, e.g. `<img slot="poster" src="...">`
https://github.com/muxinc/media-chrome/blob/74c4393cdedc7c87109e4a305070fc54cdddd540/examples/vanilla/advanced.html#L76-L86

---

yes, it's possible to hide the seekbar with setting a CSS var:

`--media-time-range-display: none;`
